scripts/setup.sh

```diff
@@ -35,6 +35,15 @@ if ! yarn install --frozen-lockfile --ignore-scripts; then
   exit ${FAILED_SETUP}
 fi
 
+# Prebuild of `microtime` requires a version of GCC with support for CXXABI_1.3.8
+# Bacon uses Amazon Linux 2 with old version of GCC which causes error
+# To fix this, don't use prebuild of `microtime`, build instead
```

Comment:
Without this an error will be thrown during unit tests at runtime:
```
  /lib64/libstdc++.so.6: version `CXXABI_1.3.8' not found (required by /root/okta/okta-auth-js/node_modules/microtime/prebuilds/linux-x64/node.napi.node)
```

Linux used in Bacon is based on CentOs, it uses old gcc 4.8.5 which can be updated only by compiling from sources (it's too long process).

Travis uses Ubuntu which has newer gcc version so doesn't have this issue